### PR TITLE
Make permissions a string as it is too big to store in Neptune as an array

### PR DESCRIPTION
**Pull request by mknoedel**
This is a breaking change for queries on this key. There are no managed questions that use this key.

Users can still query for a specific permission via `find google_iam_role with permission~={{PERMISSION_NAME}}`.

Permissions are already analyzed to create binding ALLOWS resource relationships so I don't think there will be much value in querying for a specific permission on a role.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,11 @@ and this project adheres to
 
 ## [Unreleased]
 
+### Changed
+
+- property `permissions` on `google_iam_bindings` to be a string instead of an
+  array.
```
Comment (aiwilliams) [inline CHANGELOG.md:14]: It's helpful to form these as complete sentences when bringing them into release notes. Additionally, I think it's valuable to consider expressing the impact on users.

```
- Changed `google_iam_bindings.permissions: string[]` to `google_iam_bindings.permissions: string` due to
  limitations on `string[]` property lengths in JupiterOne. Queries will need to be changed from .... to ....
```

**Review by aiwilliams — CHANGES_REQUESTED** [review]

**Comment by aiwilliams** [comment]
> Users can still query for a specific permission via find google_iam_role with permission~={{PERMISSION_NAME}}, but this query will likely be quite slow

The use of the includes matcher, `~=`, is promoted in a number of queries, so I think it's fine to recommend.

File: CHANGELOG.md (hunk 1 above)
Comment (mknoedel) [reply CHANGELOG.md]: Sounds good to me going forward to put more context into the CHANGELOG. I've kept it brief for most of my changes in the past, but I can definitely see 

> > Users can still query for a specific permission via find google_iam_role with permission~={{PERMISSION_NAME}}, but this query will likely be quite slow
> 
> The use of the includes matcher, `~=`, is promoted in a number of queries, so I think it's fine to recommend.

Works for me, I wasn't aware if there were query speed differences or not.

**Comment by mknoedel** [comment]
> > Users can still query for a specific permission via find google_iam_role with permission~={{PERMISSION_NAME}}, but this query will likely be quite slow
> 
> The use of the includes matcher, `~=`, is promoted in a number of queries, so I think it's fine to recommend.

Works for me, I wasn't aware if there were query speed differences or not.

**Review by aiwilliams — APPROVED** [review]
Looks good to me! I think it a good idea to get @austinkelleher's input on this one, as you suggested in Slack.

**Review by austinkelleher — APPROVED** [review]

**Comment by aiwilliams** [comment]
Created https://jupiterone.atlassian.net/browse/CORE-1470 to get the persister to let us know.

**Pull request merged**